### PR TITLE
Update long_range_colloquium.md -- de Leon

### PR DESCRIPTION
**Pull request by vfatemi**
Update Professor de Leon's title and abstract.

### CODE REVIEW

**Comment by akhmerov** [comment]
I'm wondering if it's possible to shorten the abstract—it's two relatively long paragraphs now. @vfatemi what do you think?

**Comment by vfatemi** [comment]
I can ask if a shorter version is possible. 

Perhaps an alternative would be to have sub-titles preceding each paragraph, like "Part 1: ..."? That way despite the overall length it's at least broken down into more chewable pieces.

File: long_range_colloquium.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ Attendance is open to everyone, but please [register](#nurit-avraham-weizmann-in
...
...eractions with external materials and spins, enabling new forms of nanoscale spectroscopy. However, NV spin coherence degrades within 100 nanometers of the surface, suggesting that diamond surfaces are plagued with ubiquitous defects. Prior work on characterizing near-surface noise has primarily relied on using NV centers themselves as probes; while this has the advantage of exquisite sensitivity, it provides only indirect information about the origin of the noise. I will describe our recent efforts to use X-ray and photoelectron spectroscopies, diffraction techniques, and morphology characterization to understand sources of noise at the diamond surface. By correlating this spectroscopic data with single spin measurements, we have been able to devise new surface processing and termination techniques to stabilize shallow NV centers within 5 nm of the surface with coherence times exceeding 100 μs [(PRX 9, 031052)](https://doi.org/10.1103/PhysRevX.9.031052). Specifically, we are able to demonstrate reversible and reproducible control over the top layer of atoms. These highly coherent, shallow NV centers will provide a platform for sensing and imaging down to the scale of single atoms.
```
Comment (vfatemi) [inline long_range_colloquium.md:30]:
```suggestion
> The nitrogen vacancy (NV) center in diamond exhibits spin-dependent fluorescence and long spin coherence times under ambient conditions, enabling applications in quantum information processing and sensing. NV centers near the surface can have strong interactions with external materials and spins, enabling new forms of nanoscale spectroscopy. However, NV spin coherence degrades within 100 nanometers of the surface, suggesting that diamond surfaces are plagued with ubiquitous defects. Prior work on characterizing near-surface noise has primarily relied on using NV centers themselves as probes; while this has the advantage of exquisite sensitivity, it provides only indirect information about the origin of the noise. I will describe our recent efforts to use X-ray and photoelectron spectroscopies, diffraction techniques, and morphology characterization to understand sources of noise at the diamond surface. By correlating this spectroscopic data with single spin measurements, we have been able to devise new surface processing and termination techniques to stabilize shallow NV centers within 5 nm of the surface with coherence times exceeding 100 μs [1]. Specifically, we are able to demonstrate reversible and reproducible control over the top layer of atoms. These highly coherent, shallow NV centers will provide a platform for sensing and imaging down to the scale of single atoms.
```

File: long_range_colloquium.md [hunk 2]
```diff
@@ -25,6 +25,12 @@ Attendance is open to everyone, but please [register](#nurit-avraham-weizmann-in
...
...ected improvements is generally applicable to many systems, and I will describe our recent efforts to tackle noise and microwave losses in superconducting qubits. Building large, useful quantum systems based on transmon qubits will require significant improvements in qubit relaxation and coherence times, which are orders of magnitude shorter than limits imposed by bulk properties of the constituent materials. This indicates that relaxation likely originates from uncontrolled surfaces, interfaces, and contaminants. Previous efforts to improve qubit lifetimes have focused primarily on designs that minimize contributions from surfaces. However, significant improvements in the lifetime of two-dimensional transmon qubits have remained elusive for several years. We have fabricated two-dimensional transmon qubits that have both lifetimes and coherence times with dynamical decoupling exceeding 0.3 milliseconds by replacing niobium with tantalum in the device [(arXiv:2003.00024)](https://arxiv.org/abs/2003.00024). We have observed increased lifetimes for many devices, indicating that these material improvements are robust, paving the way for higher gate fidelities in multi-qubit processors.
```
Comment (vfatemi) [inline long_range_colloquium.md:32]:
```suggestion
> In fact, many platforms for quantum technologies are limited by noise and loss arising from uncontrolled defects at surfaces and interfaces, including superconducting qubits, trapped ions, and semiconductor quantum dots. Our approach for correlating surface spectroscopy techniques with single qubit measurements to realize directed improvements is generally applicable to many systems, and I will describe our recent efforts to tackle noise and microwave losses in superconducting qubits [(arXiv:2003.00024)](https://arxiv.org/abs/2003.00024). 
```

File: long_range_colloquium.md (hunk 1 above)
Comment (akhmerov) [reply long_range_colloquium.md:30]: resolving: the two versions are identical.

**Pull request merged**